woocommerce/woocommerce-android
# Merge in latest changes from login library

Merges in the latest changes from the login library's `develop`. All the changes are from this PR: https://github.com/wordpress-mobile/WordPress-Android/pull/10280.

The only material change is an extra tracks event dispatched from the `SignupGoogleFragment`, which the Woo app doesn't use. However a change was also made to the `trackAnalyticsSignIn` interface method which required some minor changes to WooAndroid.

### To test

1. Make sure the project builds
2. Run through login, make sure everything is working as normal

### Update release notes

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`. (not applicable)
